docs/source/posts/2022/2022-09-28-week-14-shivam.rst

```diff
@@ -0,0 +1,45 @@
+Week 14 - Morphing is here! 
+===========================
+
+.. post:: September 28 2022
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- This week, I started with Multiple actor support in skinning, Managed to do it
+  successfully (see brainStem Model)
```

Comment:
It would be great if you either add a link to the video to see the brainStem working in the fury or directly integrate the video here itself.

Reply:
Thanks for the suggestion, I'll add it